src/util/storageVolumeTable.tsx

```diff
@@ -0,0 +1,21 @@
+export const NAME_COL = "Name";
+export const POOL_COL = "Pool";
+export const TYPE_COL = "Type";
+export const CONTENT_TYPE_COL = "Content Type";
+export const CREATED_AT_COL = "Create At";
+export const SIZE_COL = "Size";
+export const USED_BY_COL = "Used By";
+export const SNAPSHOTS_COL = "Snapshots";
+export const ACTIONS_COL = "Actions";
+
+export const COLUMN_WIDTHS: Record<string, number> = {
```

Comment:
Not sure if this approach works with all different screen widths, we should be responsive for all sizes (though we don't focus on very narrow sizes below 800px).

Reply:
I can change this to a css + media query approach? each column can get its own class name

Reply:
actually a percentage based approach should work too